src/chat.rs

```diff
@@ -1247,6 +1283,8 @@ impl Chat {
             Some(DeviceChat)
         } else if self.is_contact_request() {
             Some(ContactRequest)
+        } else if self.is_protection_broken() {
+            Some(ProtectionBroken)
```

Comment:
Hm, I just noticed that returning `false` here means that until the UIs adapt to this change, when a verification is broken, the user won't be able to send to the chat, and have no way to fix this.

Opinions on how to solve this are welcome.

Options:
- Require that all the UIs to at least treat `is_protection_broken()` like `is_contact_request()` before the next release. Disadvantage: Coordinating with all UIs is a lot of effort.
- Let `can_send()` return `true` for now, and wait until all the UIs adapt, and then introduce this change. Disadvantage: Buttons like "Video Chat" will still be shown (and work) when a chat protection is broken, even though the user is not supposed to be able to send anything before accepting.
- Let the UIs set a feature flag to enable verified 1:1 chats.